src/index.js

```diff
@@ -52,6 +52,19 @@ const instance = {
       })
   },
   /**
+   * Stops services and resolves or rejects
+   * @returns {object} promise
+   */
+  stopServices: () => {
+    const servicesStopSpinner = output.spinner('Stopping services')
+    return services.stop()
+      .then(() => servicesStopSpinner.succeed())
+      .catch((err) => {
```

Comment:
Not sure of the flow, but this error is caught and never thrown or `rejected`.  Just heads up this promise chain will truck on with a resolved state.